b2sdk/file_version.py

```diff
@@ -204,16 +205,22 @@ def from_api_response(self, file_version_dict, force_action=None):
         )
 
     def from_response_headers(self, headers):
+        file_info = {}
+        prefix_len = len(FILE_INFO_HEADER_PREFIX_LOWER)
+        for header_name, header_value in headers.items():
+            if header_name[:prefix_len].lower() == FILE_INFO_HEADER_PREFIX_LOWER:
+                file_info_key = header_name[prefix_len:]
+                file_info[file_info_key] = header_value
         return FileVersion(
             api=self.api,
             id_=headers.get('x-bz-file-id'),
             file_name=headers.get('x-bz-file-name'),
             size=headers.get('content-length'),
```

Comment:
if we download with `Content-Range` header this would be the length of selected range - I don't like this even for `DownloadVersion` we should check if `Content-Range` response header is available and parse `size` from this - and add `content_length` for `DownloadVersion` (`size == content_length` if range is not used)

Reply:
working on it